src/functions-and-arrays.js

```diff
@@ -1,19 +1,60 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(a,b) {
+  if (a>b){
+    return a;
+  }
+  else if(a<b) {
+    return b;
+  }
+  else {
+    return a || b;
+  }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arrayOfWords) {
+  let longestWord = "";
+
+  if(arrayOfWords == ""){
```

Comment:
Here you compare the array with an empty string. To check if the array is empty:
`arrayOfWords.length === 0`